src/main/java/org/fcrepo/migration/validator/Driver.java

```diff
@@ -112,6 +112,10 @@
                                       "OCFL repositories. This validation is always disabled if a PID File is used.")
     private boolean checkNumberOfObjects;
 
+    @CommandLine.Option(names = {"--delete-inactive", "-I"}, order = 18,
```

Comment:
maybe change the flag to --inactive-as-deleted  - just to be clear that no deletion will occur.

Reply:
@dbernstein This has been updated to change the name for the cli option